docs/docs/how-tos/nebari-gcp.md

```diff
@@ -4,20 +4,20 @@ title: Deploy Nebari on GCP
 description: A basic overview of how to deploy Nebari on GCP.
 ---
 
-## Introduction
-
-This guide is to help first-time users set up a Google Cloud Platform account specifically for the purpose of using and deploying Nebari at a production scale. In this guide we
-will walk you through the following steps:
-
 - [Introduction](#introduction)
 - [Sign up for Google Cloud Platform](#sign-up-for-google-cloud-platform)
-- [Set up the `gcloud` CLI](#set-up-the-gcloud-cli)
 - [Authentication](#authentication)
+- [Required GCP APIs](#required-gcp-apis)
 - [Initializing Nebari](#initializing-nebari)
 - [Deploying Nebari](#deploying-nebari)
 - [Destroying Nebari](#destroying-nebari)
 
-For those already familiar with Google Cloud Platform and `gcloud`, feel free to skip this first step and jump straight to the [Nebari authentication](#authentication) section of
+## Introduction
+
+This guide is to help first-time users set up a Google Cloud Platform account specifically for the purpose of using and deploying Nebari at a production scale. In this guide we
+will walk you through the following steps:
```

Comment:
Since this got moved, this sentence needs reworded.

Reply:
https://github.com/nebari-dev/nebari-docs/issues/540 should track any work on refactoring these

Reply:
@dcmcand I moved the Table of Contents below this paragraph to make it more readable and consistent with what we have for other cloud providers.

@viniciusdc great, we can do further improvements later under the scope of that issue.